### ISSUE TITLE
quickstart: set up new context?

### ISSUE THREAD

**Issue by pinheadmz** [bitcoin-dev-project/warnet#542]
Got this trying to `warnet deploy`:
```
Exception: error: current-context must exist in order to minify
```

then realized I had no contexts:
```
n$ k config get-contexts
CURRENT   NAME   CLUSTER   AUTHINFO   NAMESPACE
```

I actually dont know what to do next -- something with minikube maybe?

**Comment by pinheadmz** [comment]
Might be fixed by #509 I did pip install warnet so im not on master

**Comment by pinheadmz** [comment]
OK on master it does direct user to install minikube but would be cool if it also ensured `minikube start` which checks the backend driver like docker is running as well

**Comment by pinheadmz** [comment]
closed by #548
